api/controllers/game/spectate-leave.js

```diff
@@ -0,0 +1,20 @@
+module.exports = async function (req, res) {
+  const gameId = req.session.spectating;
+  const spectator = await userService.findUser({ userId: req.session.usr });
+
+  try {
+    await UserSpectatingGame.update({ gameSpectated: gameId, spectator: spectator.id }).set({
+      activelySpectating: false,
+    });
+
+    const fullGame = await gameService.populateGame({ gameId: gameId });
+    Game.publish([fullGame.id], {
+      change: 'spectatorLeft',
+      game: fullGame,
+    });
+    return res.ok(fullGame);
+  } catch (err) {
+    console.log(err);
```

Comment:
We don't log errors to the server console b/c no one reads them. The right thing to do is to set up error logging, which we have an issue for but haven't completed. For now we can just nix this line
```suggestion
```

Reply:
Oops! Yea haha this was just for myself. Forgot to delete them lol